lib/testutils/testutils.go

```diff
@@ -52,3 +53,49 @@ func (rw *NoopRW) Read(p []byte) (n int, err error) {
 func (rw *NoopRW) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
+
+type NoopCloser struct{}
```

Comment:
`ioutil.NopCloser(nil)` but we can also have our own, skips the nil reader baggage 🤷